### PR TITLE
Student work type

### PR DESCRIPTION
**Pull request by jamesvanmil**

### CODE REVIEW

**Comment by jamesvanmil** [comment]
Ready for review - has a sibling PR on scholar_uc to close the issue

File: app/views/curation_concern/student_works/_form_required_information.html.erb [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+                        collection: work_types_for_student_works,
+                        label_method: ->(value) do 
+                          if value == "Generic Work" 
+                            "Other"
+                          else 
+                            value 
+                          end
+                        end %>
+          </div>
+          <%= render 'form_input_help_block', help_text: I18n.t('sufia.work.input_field.title.help', work_type: 'work') %>
```
Comment (scherztc) [inline app/views/curation_concern/student_works/_form_required_information.html.erb]: Point this to a work_type.help locale.

File: config/locales/sufia.en.yml [hunk 2]
```diff
@@ -99,3 +105,9 @@ en:
             Document:  "A PDF is preferred. Hold Shift or Command to select multiple files" 
             Dataset:   "Hold Shift or Command to select multiple files"
             Video:  "Hold Shift or Command to select multiple files"
+            StudentWork:  "Hold Shift or Command to select multiple files"
+    student_work:
+      input_field:
+        description: 
+          help:         "<p>Enter a summary of your %{work_type}. Please include any presentations or awards assocaited with this %{work_type} here. There is no character limit for this field.</p>"
+
```
Comment (scherztc) [inline config/locales/sufia.en.yml:115]: You may want to put the work_type information here.

**Comment by scherztc** [comment]
screen shot 2016-05-09 at 10 28 12 am

BLANK work type does creates a mismatch with the facet count.  This is minor, but I still feel that it should be a required field.

**Comment by jamesvanmil** [comment]
+1 - we have the generic work/other bucket, so it seems like it wouldn't be too much of a nuisance to make it required. My only concern is that it's harder on that page to provide enough context about what the best fit is.

**Comment by scherztc** [comment]
Maybe instead of required we start with a Generic Work default on the form.

**Comment by jamesvanmil** [comment]
@ntallman - what do you think about setting a default?

**Comment by ntallman** [comment]
@jamesvanmil @scherztc I'm not crazy about defaulting to generic work. I'd rather leave it blank and make it a required field. Seems like the former option could result in a lot of generic works that are better as other work types.

**Pull request merged**